HW3/src/js/task1.js

```diff
@@ -0,0 +1,11 @@
+let x = prompt('Введите сумму');
```

Comment:
use a better name for variables to better understand what they are holds

Reply:
+